source/aggregation-pipeline-builder.txt

```diff
@@ -112,35 +112,31 @@ your chosen view mode:
       .. procedure:: 
          :style: normal
 
-         .. step:: In the aggregation pipeline pane, ensure the 
+         .. step:: Select the :guilabel:`Stage` view.
+            
+            In the aggregation pipeline pane, ensure the 
```

Comment:
[suggestion] Consider, "be sure the..."

Reply:
Likely opting to stick with "ensure" based on recommendation from the [style guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#e:~:text=clients%20and%20devices.-,ensure,-Use%20ensure%20to)